From 2b407ad4e797696c776e125dcdd441dd8fbcf80f Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:09 -0400
Subject: [ethernet] igb: Cleanups to fix pointer location error

Message-id: <1409062359-13063-52-git-send-email-sassmann@redhat.com>
Patchwork-id: 89169
O-Subject: [RHEL7.1 PATCH 51/81] igb: Cleanups to fix pointer location error
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Carolyn Wyborny <carolyn.wyborny@intel.com>

This patch fixes ERROR:POINTER_LOCATION from checkpatch file check.

Signed-off-by: Carolyn Wyborny <carolyn.wyborny@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c4917c6ffe31d7bd3baacb80a87107319df52979)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/e1000_82575.c b/drivers/net/ethernet/intel/igb/e1000_82575.c
index 49976bf..a83fcad 100644
--- a/drivers/net/ethernet/intel/igb/e1000_82575.c
+++ b/drivers/net/ethernet/intel/igb/e1000_82575.c
@@ -525,7 +525,7 @@ out:
 static s32 igb_get_invariants_82575(struct e1000_hw *hw)
 {
  struct e1000_mac_info *mac = &hw->mac;
- struct e1000_dev_spec_82575 * dev_spec = &hw->dev_spec._82575;
+ struct e1000_dev_spec_82575 *dev_spec = &hw->dev_spec._82575;
  s32 ret_val;
  u32 ctrl_ext = 0;
  u32 link_mode = 0;
-- 
1.7.1